contributors/design-proposals/cli/preserve-order-in-strategic-merge-patch.md

```diff
@@ -228,7 +228,7 @@ If an old kubectl sends a old patch to a new server,
 the server will honor the order in the list as mentioned above.
 The behavior is a little different from before but is not a breaking change.
 
-If a new kubectl sends a new patch to an old server, the server doesn't recognise the parallel list and will drop it.
+If a new kubectl sends a new patch to an old server, the server doesn't recognize the parallel list and will drop it.
```

Comment:
Please remove this change. "recognise" is the British English spelling of the term, and either is acceptable in our documentation (at the author's discretion)

Reply:
OK, It's removed now.
Thanks!